src/components/Tx/SendTx/CurrencySelect.jsx

```diff
@@ -0,0 +1,223 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Identicon from '../../Identicon'
+
+export default class CurrencySelect extends Component {
+  static displayName = 'CurrencySelect'
+
+  static propTypes = {
+    etherWallet: PropTypes.shape({
+      balance: PropTypes.string,
+      symbol: PropTypes.string,
+      address: PropTypes.string,
+      name: PropTypes.string
```

Comment:
The user should not have to pass in 'ETHER' every time for `name` and `symbol`. This value should never change, so there's no reason to give the user the ability to goof it up.